personal-practice/diceMain.java

```diff
@@ -0,0 +1,13 @@
+import java.awt.*;
+import java.lang.Math;
+import java.util.Random;
+
+public class diceMain {
+    
+    public static void main(String[] args) {
+        Dice dice = new Dice();
+        System.out.println(dice.roll());
+        System.out.println(dice.roll());
+        System.out.println("Previous: " + dice.previousRoll);
```

Comment:
Suggestion: add a getter method instead of accessing the field directly from outside.